templates/docs/examples/patterns/navigation/_script-sliding.js

```diff
@@ -106,30 +106,46 @@ const initNavigationSliding = () => {
   toggles.forEach(function (toggle) {
```

Comment:
Some logic responsible for closing the navigation items (ex. removing and adding the 'is-collapsed' class) shouldn't be included in the event listen for the buttons themselves as this isn't the sole way to close the dropdowns

Reply:
Good point!